templates/mongodb.conf.erb

```diff
@@ -180,3 +180,8 @@ syslog = <%= @syslog %>
 <% if @quiet -%>
 quiet = <%= @quiet %>
 <% end -%>
+<% if @ssl -%>
```

Comment:
Yes, `if @ssl and @ssl_key and @ssl_ca` because if someone just passes `ssl => true` then the template will still contain blank lines. Perhaps instead the manifest should have:
```puppet
  if $ssl {
    validate_string($ssl_key, $ssl_ca)
  }
```